### PR TITLE
Gem: remove frank related code and tests

### PR DESCRIPTION
**Pull request by polyakrecords**
[VSTS](https://msmobilecenter.visualstudio.com/Mobile-Center/Krukow-M1-Team/_workitems/edit/39773)

- Removed Frank related code and tests

### CODE REVIEW

File: calabash-cucumber/spec/integration/launcher/inject_dylibs_spec.rb [hunk 1]
```diff
@@ -1,36 +0,0 @@
-
```
Comment (jmoody) [inline calabash-cucumber/spec/integration/launcher/inject_dylibs_spec.rb]: This change should be in a separate change set.

You can push this change directly to /develop.

**Comment by polyakrecords** [comment]
@jmoody Force pushed and rebased

**Comment by jmoody** [comment]
@polyakrecords 

We need to switch to Jenkins 2.0 CI for this project.

Can you create a VSTS issue under Xcode 9.4 beta 4 support?

**Comment by polyakrecords** [comment]
Jenkins test this please

**Pull request merged**